shared/address_explorer.py

```diff
@@ -197,7 +187,15 @@ def generate_address_csv(path, addr_fmt, n):
         return
 
     path, addr_fmt = picked
+    chain = chains.current_chain()
+    def get_address_at_idx(idx):
+        with stash.SensitiveValues() as sv:
```

Comment:
This change will impact performance heavily. `stash.SensitiveValues()` context manager takes a long time to start up because it talks via bootrom to the SE. Moving it outside the loop like this will be terrible (on real hardware).

Reply:
Makes sense. I could try refactoring it to `get_addresses(from, to)` so that all the addresses for a given screen / text file backup will be calculated in the same `stash.SensitiveValues()` context. Sound good?